llvm/test/CodeGen/RISCV/rvv/rv32-spill-zvlsseg.ll

```diff
@@ -167,8 +167,8 @@ define <vscale x 4 x i32> @spill_zvlsseg_nxv4i32(ptr %base, i32 %vl) nounwind {
 ; SPILL-O0-NEXT:    csrr a2, vlenb
 ; SPILL-O0-NEXT:    slli a2, a2, 1
 ; SPILL-O0-NEXT:    sub sp, sp, a2
-; SPILL-O0-NEXT:    vsetvli zero, a1, e32, m2, ta, ma
 ; SPILL-O0-NEXT:    # implicit-def: $v8m2_v10m2
+; SPILL-O0-NEXT:    vsetvli zero, a1, e32, m2, tu, ma
```

Comment:
Potentially a case where hasUndefinedMergeOp is failing, causing the tu policy.

Reply:
Oh I see this is on -O0 where we don't have the register coalescer. In that case I presume it's because there's a COPY somewhere that's getting in the way of the undef flag. I think we can safely ignore this though for O0.